WalletWasabi.Fluent/ViewModels/Wallets/CoinJoinStateViewModel.cs

```diff
@@ -76,13 +77,15 @@ private enum Trigger
 
 	private readonly MusicStatusMessageViewModel _finishedMessage = new() { Message = "No balance to coinjoin" };
 
-	private DateTimeOffset _autoStartTime;
+	private TimeSpan _autoStartTime;
 	private DateTimeOffset _countDownStarted;
 
 	public CoinJoinStateViewModel(WalletViewModel walletVm, IObservable<Unit> balanceChanged)
```

Comment:
This is created after the wallet loads successfully?

Reply:
yes, this viewmodel is created only once when wallet loads.